vuework02/src/views/Step2.vue

```diff
@@ -42,13 +42,34 @@ export default {
 	},
     showQuestion2() {
 	this.isActive2 = !this.isActive2
-  }
+  },
+    addAnswer1(event) {
+      if(event.target.value==='はい') {
```

Comment:
`===`の前後にはスペースをあけましょう。
```suggestion
      if(event.target.value === 'はい') {
```
他の部分でも同様です。